components/ILIAS/Course/classes/Preview/class.ilCourseMailTemplateTutorContextPreview.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+class ilCourseMailTemplateTutorContextPreview extends ilCourseMailTemplateTutorContext
+{
+    public const ID = 'crs_context_tutor_manual_preview';
+    public const DEFAULT_COURSE_TITLE = "preview_crs_title";
+    public const DEFAULT_COURSE_STATUS = "preview_crs_status";
+    public const DEFAULT_COURSE_MARK = "preview_crs_mark";
+    public const DEFAULT_COURSE_TIME_SPENT = "3671";
```

Comment:
This file does not belong to "my" components, but I suggest:

```suggestion
    public const string ID = 'crs_context_tutor_manual_preview';
    public const string DEFAULT_COURSE_TITLE = "preview_crs_title";
    public const string DEFAULT_COURSE_STATUS = "preview_crs_status";
    public const string DEFAULT_COURSE_MARK = "preview_crs_mark";
    public const string DEFAULT_COURSE_TIME_SPENT = "3671";
```